src/WCSimDetectorConfigs.cc

```diff
@@ -29,13 +29,13 @@ void WCSimDetectorConstruction::SetSuperKGeometry()
   WCPMTRadius = PMT->GetRadius();
   WCIDDiameter          = 33.6815*m; //16.900*2*cos(2*pi*rad/75)*m; //inner detector diameter
   WCIDHeight            = 36.200*m; //"" "" height
-  WCBarrelPMTOffset     = 0.0715*m; //offset from vertical
+  WCBarrelPMTOffset     = WCPMTRadius; //offset from vertical
```

Comment:
That surprises me, because the drawings shown in this pull request illustrate that there is an overlap when I was running the default SuperK geometry and why the change is necessary.
Did you run with both checkOverlaps == true?